### ISSUE TITLE
request: support Trajan Pro for card names & types

### ISSUE THREAD

**Issue by bszonye** [sumpfork/dominiontabs#370]
Official Dominion cards use four typefaces: Trajan Pro Bold for the card names and types, Minion Pro Bold for the cost numbers, and Times Roman & Bold for the rules text. Trajan is not as widely available as the other two, although it's included with all Adobe Creative Cloud subscriptions, so anybody with a Photoshop license will have access. There are also downloads available at fontsgeek like there are for Minion Pro. It would be nice if folks with Trajan installed could use it for a closer match to the cards.

I have a working implementation in my fork that uses Trajan Pro if available and Minion Pro if not. It's somewhat complicated by the fact that Trajan is a small caps typeface and Minion isn't, so all of the code that currently simulates small caps needs an extra code path to handle the case where it's built into the font. Trajan also has very different metrics, with a caps height of 0.750 and x-height of 0.638, whereas the simulated small caps using Minion have a caps height of 0.651 and an x-height of 0.5425, so integrating the new font also means making a bunch of small layout adjustments. I'm still fine-tuning things there.

This came up while I was working on a bigger project, so my work in progress is currently entangled with the other features. If there's a strong demand, I could factor out the Trajan work like I have with my other PRs, but it's significantly more effort than those were. I'll create an issue for the other features and send a PR when the work is more stable.

**Comment by bszonye** [comment]
I found that this is a good search list for Trajan Bold. Adobe has retired the original font family and replaced it with Trajan Bold 3, which has some additional weights. For Dominion cards, you'll want the semibold.
```
TrajanPro-Bold.ttf
TrajanPro3-Semibold.ttf
Trajan Pro Bold.ttf
```

**Comment by bszonye** [comment]
Today I learned that the box art uses CharlemagneStd-Bold for expansion names, so that's another potential enhancement.

**Comment by bszonye** [comment]
I've been learning a lot about Dominion typography, ha. It seems that the card costs also use a slightly different font, Minion Std Black. The current choice, Minion Pro Bold, is a very good substitute, but the card font is a bit darker and chunkier. I'm collecting all of these as I find them and have implemented a better fallback system.

**Comment by bszonye** [comment]
Resolved in #433.
